cmd/samCoverage/samCoverage.go

```diff
@@ -0,0 +1,103 @@
+// Command Group: "SAM Tools"
+
+// Calculates genome coverage as the quotient of aligned bases in a sequencing dataset to the total length of ungapped genomic regions in the reference genome
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+
+	"github.com/vertgenlab/gonomics/dna"
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/numbers"
+	"github.com/vertgenlab/gonomics/numbers/fit"
+	"github.com/vertgenlab/gonomics/sam"
+)
+
+type Settings struct {
+	SamFileName   string
+	OutFile       string
+	CountNinDepth bool
+	Verbose       int
+}
+
+// samCoverage calculates genome coverage as the quotient of aligned bases in a sequencing dataset to the total length of ungapped genomic regions in the reference genome.
+func samCoverage(s Settings) {
+	out := fileio.EasyCreate(s.OutFile)
+	fmt.Fprintf(out, "Coverage\tPileups\tGroup\n")
```

Comment:
There's an unhandled error here. Consider if the outFile is deleted while it's open, or if for some reason it can't be written to. In this case, fmt.Fprintf would return an error, and we'd want to catch if it were non-nil. We can change to something like this:
`_, err = fmt.Fprint(out, "Coverage\tPileups\tGroup\n")`
`exception.PanicOnErr(err)`